src/main/java/org/primeframework/mvc/sampleapp/action/FormAction.java

```diff
@@ -0,0 +1,44 @@
+package org.primeframework.mvc.sampleapp.action;
+
+import com.google.inject.Inject;
+import io.fusionauth.http.HTTPValues;
+import org.primeframework.mvc.action.annotation.Action;
+import org.primeframework.mvc.message.MessageStore;
+import org.primeframework.mvc.message.MessageType;
+import org.primeframework.mvc.message.SimpleFieldMessage;
+import org.primeframework.mvc.validation.ValidationMethod;
+
+@Action
+public class FormAction extends BaseAction {
+    private final MessageStore messageStore;
+
+    public String result;
+    public String yourName;
+
+    @Inject
+    public FormAction(MessageStore messageStore) {
+        this.messageStore = messageStore;
+    }
+
+    public String get() {
+        return "success";
+    }
+
+    public String post() {
+        if (yourName != null && yourName.contains("crash")) {
+            throw new RuntimeException("we have a problem");
+        }
```

Comment:
If we want to show an example of real world workflows, we would generally never intentionally throw a runtime exception like this. 

In prime, we always want to throw exceptions that extend `ErrorException` if you want a predictable result. 

If the intent is just to have an exception in your example that always throws an un-expected exception, should we jsut create a new action named `ThrowsExcceptionAction` (`/throws-exception`)?

Reply:
This is mimicing a problem with a service or some backend code.